### ISSUE TITLE
Fine tuning on a pretrained model can lead to poor performance

### ISSUE THREAD

**Issue by lyc200522** [mlfoundations/open_clip#740]
When I used the laion400M dataset on ViT-B-16_ Laion2b_ S34b_ b88k when fine-tuning it, I found that the zero-shot performance of Imagenet decreased by more than 20% with training. And the more you train, the more performance will decrease

**Comment by gabrielilharco** [comment]
Unfortunately this is a common problem with fine-tuning. We explore this in [this paper](https://arxiv.org/abs/2208.05592), and propose a simple way of mitigating this: interpolating between the weights of the original and fine-tuned models. I'm curious if it would work for you as well! There's code here https://github.com/mlfoundations/patching

**Comment by mittski** [comment]
@gabrielilharco I just saw mlfoundations/patching, and it looks very similar to the [wise-ft](https://github.com/mlfoundations/wise-ft) repo  ... seeing as you're an author on both papers, could you explain the difference between the two repos/papers? Is one just another exploration of the same premise in a different implementation? To a ml-amateur like myself, I can't see a significant difference between them

**Comment by gabrielilharco** [comment]
Hey @mittski, indeed they are very similar and the central idea (averaging weights of fine-tuned and pre-trained models) is the same on both papers. The main practical difference is that in wise-ft we create new classification heads when fine-tuning, and in the patching paper, we don't introduce any weights (we keep the classification heads frozen so the model remains "open-vocabulary"). The focus of wise-ft was to fine-tune on a single task, while the focus of patching is to learn new tasks without reducing accuracy on tasks where the model is already good

**Comment by mittski** [comment]
Ah, I see. I should have read both the papers more closely, thanks for the clarification

**Comment by EIFY** [comment]
No guarantee that it works here, but perhaps it's worth mentioning that the SigLIP paper reports that it helps a lot to disable weight-decay of the pre-trained backbone during fine-tuning 🤔

https://arxiv.org/abs/2303.15343
<img width="320" alt="Screenshot 2023-11-15 at 6 11 23 PM" src="https://github.com/mlfoundations/open_clip/assets/2584418/ef590f94-62d9-400d-9da5-c705403d475b">

**Comment by lyc200522** [comment]
My task is focused on the CV aspect, and I have found that only fine-tuning the visual module performance of CLIP will only result in slight changes, compared to fine-tuning the entire CLIP.
